pages/clubs.jsx

```diff
@@ -409,6 +409,7 @@ var ModalAddOrChangeYourClub = React.createClass({
              </div>
            : null}
           {this.renderValidationErrors()}
+          <p>We're full! Let us know if you'd like to be considered for the next round.</p>
```

Comment:
I've not been consistent about this and I just saw the copy on [line 493](https://github.com/mozilla/teach.webmaker.org/pull/1096/files#diff-22389187e0af4479c85f04a20bd51e17R493) ... but should I change `'` into `&lsquo;`?